PoGoEmulator/PoGoEmulator/Extensions.cs

```diff
@@ -135,18 +135,19 @@ public static void WriteProtoResponse(this NetworkStream ns, HttpStatusCode stat
                 StatusCode = (int)statusCode,
                 Error = errorMessage
             };
-            ns.WriteHttpResponse(responseToUser.ToByteString().ToArray());
+            ns.WriteHttpResponse(responseToUser.ToByteString());
         }
 
-        public static void WriteHttpResponse(this NetworkStream ns, byte[] body)
+        public static void WriteHttpResponse(this NetworkStream ns, ByteString body)
```

Comment:
there is a problem occur when response data sends,
expected response:
`HTTP/1.1 200 OK
Date: Sun, 28 May 2017 02:24:32 GMT
Connection: keep-alive
Content-Length: 56

5f??? ÛO~P!pgorelease.nianticlabs.com/custom:»^èçÄ+`

current response:
`HTTP/1.1 200 OK
Connection: keep-alive
Server: C#.NET
Date: Sun, 28 May 2017 03:32:21 GMT
Content-Length: 56

5��������S!pgorelease.nianticlabs.com/custom:ȱ���+`

length is okey but string ecode is problem, (Encoding.utf8 are Encoding.default are tried but same issue)